microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/OpenApiDeploymentUrlProvider.java

```diff
@@ -29,4 +29,16 @@ public static String composeDefaultOpenApiUrl() {
                 OpenApiTestConstants.DEFAULT_HOST_NAME,
                 OpenApiTestConstants.DEFAULT_ENDPOINT_PORT);
     }
+
+    /**
+     * Builds OpenAPI endpoint URL for HTTPS access
+     *
+     * @return
+     */
+    public static String composeHttpsOpenApiUrl() {
```

Comment:
This isn't needed/wanted, you can obtain it from ARQ

deployment url - https://docs.jboss.org/author/display/ARQ/ArquillianResource+injection
```java
@ArquillianResource
private URL baseURL;
```

management interface
```java
@ContainerResource
ManagementClient managementClient;

managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort()
```
https://github.com/wildfly/wildfly/blob/master/testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/metrics/base/MicroProfileMetricsBaseTestCase.java#L57
https://github.com/wildfly/wildfly/blob/master/testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/metrics/MetricsHelper.java#L34
https://github.com/wildfly/wildfly-arquillian/blob/master/common/src/main/java/org/jboss/as/arquillian/container/ManagementClient.java

Reply:
Thanks for the pointers, fixing.